### ISSUE TITLE
Deal with Assertion in TSPUnarchiver

### ISSUE THREAD

**Issue by masaccio** [masaccio/numbers-parser#40]

**Comment by masaccio** [comment]
Example assertion:

```
#Assert *** Assertion failure #0: -[TSPUnarchiver validateReferenceToObjectIdentifier:objectClass:isWeak:validateStrongReferences:selector:weakSelector:] /Library/Caches/com.apple.xbs/Sources/iWorkDependenciesMacOS/iWorkDependenciesMacOS-7034.0.86/shared/persistence/src/TSPUnarchiver.mm:704 Object [TSTTableTile-10000011] is not strongly referenced from message [TST.TableModelArchive-904538] message type 6001. Use TSPReadWeakLazyReferenceMessage<TSTTableTile>(unarchiver, message, completion) instead.
```

**Comment by SheetJSDev** [comment]
That particular assertion was resolved in 3.4.2 and the root cause is understood (a missing reference)
